src/slack_interaction_handler.ts

```diff
@@ -348,15 +355,17 @@ export async function handleVolunteerUpdate({
 
   // Post a message in the voter thread recording this status change.
   await SlackApiUtil.sendMessage(
-    `*Operator:* Volunteer changed to *${selectedVolunteerSlackUserName}* by *${originatingSlackUserName}* at *${specialSlackTimestamp}*.`,
+    selectedVolunteerSlackUserName
+      ? `*Operator:* Volunteer changed to *${selectedVolunteerSlackUserName}* by *${originatingSlackUserName}* at *${specialSlackTimestamp}*.`
+      : `*Operator:* Volunteer claim released by *${originatingSlackUserName}* at *${specialSlackTimestamp}*.`,
```

Comment:
Is `originatingSlackUserName` null here in the release claim case?

Reply:
no, originatingSlackUserName is the person who clicked the button (not necessarily the old assigned volunteer)

Reply:
I wasn't certain if the payload's value for the drop-down would reliably have the old volunteer so i left it off the message. I'm not sure it is that important.. you can look in the history to see who the assigned person was before

Reply:
Right thanks! No problem, I don't think there's a need to have the message include the old volunteer claim.